management/status_checks.py

```diff
@@ -805,11 +806,11 @@ def check_miab_version(env, output):
 		latest_ver = get_latest_miab_version()
 
 		if this_ver == latest_ver:
-			output.print_ok("Mail-in-a-Box is up to date. You are running version %s." % this_ver)
+			output.print_ok("Mail-in-a-Box with quota support is up to date. You are running version %s." % this_ver)
```

Comment:
This is not needed in the PR right? I understand this is useful when running your fork, but not in the 'official' MIAB.